### PR TITLE
Docs description improvement

### PR DESCRIPTION
**Pull request by kolya182**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ const constants = require('bip44-constants')
...
-  const constant = constants[coin]
+Object.keys(constants).forEach(coinSymbol => {
+  const constant = constants[coinSymbol]
 
   // ...
-  console.log(coin, constant)
+  console.log(coinSymbol, constant)
 })
 
 console.log(constants['Litecoin'])
```
Comment (fanatid) [inline README.md]: In this case this also should be fixed :)

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules/
+package-lock.json
```
Comment (fanatid) [inline .gitignore]: I'm not sure that we really need this new line

Comment (kolya182) [reply .gitignore]: `npm i` creates it, some people may be confusing including in PR or not, it will not hurt

Comment (fanatid) [reply .gitignore]: yarn also create lock file, text editors create temporary files -- but we did not have this in gitignore..

Comment (kolya182) [reply .gitignore]: But you don't have `yarn install` to ignore yarn lock file right ?. This is what i faced when first worker with repo.

Comment (fanatid) [reply .gitignore]: I use yarn, but not include lock to repo, because it was not included historically. I cherry-picked fixes for README.

**Pull request closed without merging**